DBM-ChamberOfAspects/Ruby/Zarithrian.lua

```diff
@@ -10,44 +10,65 @@ mod:RegisterEvents(
 	"SPELL_CAST_START",
 	"SPELL_AURA_APPLIED",
 	"SPELL_AURA_APPLIED_DOSE",
+	"SPELL_AURA_REMOVED",
 	"CHAT_MSG_MONSTER_YELL"
 )
 
 local warningAdds				= mod:NewAnnounce("WarnAdds", 3, 74398)
 local warnCleaveArmor			= mod:NewStackAnnounce(74367, 2, nil, "Tank|Healer")
+local warnFearSoon              = mod:NewSoonAnnounce(74384, nil, nil, nil, 2, 2)
```

Comment:
```suggestion
local warnFearSoon              = mod:NewSoonAnnounce(74384, 2, nil, nil, nil, nil, nil, 2)
```
maybe like this? idk what you wanted to achieve with 2, 2 args